README.md

```diff
@@ -1,7 +1,7 @@
 # We are Nmstate!
 A declarative network manager API for hosts.
 
-[![Unit Test Status](https://travis-ci.org/nmstate/nmstate.png?branch=master)](https://travis-ci.org/nmstate/nmstate)
+[![Unit Test Status](https://travis-ci.com/nmstate/nmstate.png?branch=master)](https://travis-ci.com/nmstate/nmstate)
```

Comment:
Would you mind also removing the `Unit` from the text since it is not only unit tests anymore?

Reply:
Done